pages/styles/portfolio.css

```diff
@@ -0,0 +1,163 @@
+#grid-container {
+    display: grid;
+    grid-template-columns: repeat(4, 300px);
+    grid-template-rows:  250px 250px;
+    justify-items: center;
+    grid-template-areas:
+    "header header header header"
+    "content content1 content2 content3"
+}
+
+#grid-container {
+    border: 5px solid #fde2f0;
+    border-radius: 5px;
+    background-color: white;
+}
+
+#item1 {
+    grid-column: span 3;
+    grid-area: header;
+} 
+
+#item2 {
+    grid-column: span 1;
+    grid-row: span 1;
+    grid-area: content;
+}
+
+#item3 {
+    grid-column: span 1;
+    grid-row: span 1;
+    grid-area: content1;
+}
+
+#item4 {
+    grid-column: span 1;
+    grid-row: span 1;
+    grid-area: content2;
+}
+
+#item5 {
+    grid-column: span 1;
+    grid-row: span 1;
+    grid-area: content3;
+}
```

Comment:
While using IDs here is fine, this does end up repeating these rules 4 times, and we'll have to keep repeating as you add more projects.

In this case, we have multiple elements that are all the same so rather than an ID, I recommend using a class, perhaps one named `item` (well, perhaps a more informative name than `item`...).

This does change how you would handle your `#grid-container` rule but it gets rid of a lot of repetition:

```css
#grid-container {
    display: grid;
    grid-template-columns: repeat(4, 300px);
    grid-template-rows:  250px 250px;
    justify-items: center;
    grid-template-areas:
    "header header header header"
    "content content content content"
}

.item {
    grid-column: span 1;
    grid-row: span 1;
}

/* ... */

.item a {
    background: #c6fbf9;
    padding: 35px 20px;
    text-decoration: none;
    font-weight: bold;
    font-size: 20px;
    color: #faffdb;
    display: inline-block;
    margin-top: -15px;
    margin-bottom: 160px;
    border-radius: 8px;
}

item a:hover{
    color: #ffebe0;
}
```

I gave this a try after cloning your repro and it seemed to look the same, so it would be a good way to cut on repetition here.